### DISCUSSION TITLE
Existing batch after LSI and Harmony

### DISCUSSION THREAD

**Discussion by chen-zhan** [GreenleafLab/ArchR#1843] [General]
Hi, thanks for providing such an excellent software.
After I run LSI and harmony, I was confussed when I saw the picture which I draw(as the pdf file showed). There are a cluster which is with many batch library.
[03.Plot-after-cluster.pdf](https://github.com/GreenleafLab/ArchR/files/10727822/03.Plot-after-cluster.pdf)
I try to follow the 4.2 guidance(https://www.archrproject.com/bookdown/iterative-latent-semantic-indexing-lsi.html), But it seem no use.
`    #reducedDims
    proj <- addIterativeLSI(
        ArchRProj = proj,
        useMatrix = "TileMatrix", 
        name = "pp_IterativeLSI", 
        iterations = 8, #up
        clusterParams = list( #See Seurat::FindClusters
            resolution = c(0.1, 0.2, 0.4, 0.4, 0.4, 0.4, 0.4), 
            sampleCells = 10000, 
            n.start = 10
        ), 
        force = FALSE,
        varFeatures = 25000, #down
        dimsToUse = 1:30
    )
    
    #Remove Batch
    proj <- addHarmony(
        ArchRProj = proj,
        reducedDims = "pp_IterativeLSI",
        name = "pp_Harmony",
        force = FALSE,
        lambda = 0.6,
        epsilon.harmony = -Inf,
        corCutOff = 0.7,
        max.iter.harmony = 10,
        groupBy = "Sample"
    )`

[ArchR-addIterativeLSI-12f6d30188847-Date-2023-02-13_Time-15-56-15.log](https://github.com/GreenleafLab/ArchR/files/10727850/ArchR-addIterativeLSI-12f6d30188847-Date-2023-02-13_Time-15-56-15.log)

I would be very grateful if you have any suggestions on this issue.

**Comment by rcorces** [comment]
Sorry - I dont provide specific advice for individual analyses on this forum. Your only real option is read the documentation on `harmony::HarmonyMatrix` to see if there are other parameters or usage tweaks you can use on your particular dataset
